src/components/LocationPage/SocialButtons.tsx

```diff
@@ -77,6 +75,21 @@ const SocialButtons = ({
           </CopyLinkButton>
         </SocialShareButton>
       </CopyToClipboard>
+      <SocialShareButton variant="contained" color="#007fb1">
+        <SocialButton
+          disableRipple
+          disableFocusRipple
+          disableTouchRipple
+          onClick={() => setShowEmbedPreviewModal(true)}
```

Comment:
Consider moving the `disableRipple`, etc attributes to the styled component, that way it will be consistent across usages

```tsx
export const SocialButton = styled(Button).attrs(props => ({
  disableElevation: true,
  disableRipple: true,
  // ...
}))`
  width: 60px;
  height: 42px;
  // ...
`;

```